api_docs/actions.json

```diff
@@ -71,18 +72,13 @@
         "description": [],
         "children": [
           {
+            "id": "def-server.asSavedObjectExecutionSource.$1",
             "type": "Object",
             "label": "source",
             "isRequired": true,
             "signature": [
               "Pick<",
-              {
-                "pluginId": "core",
-                "scope": "common",
-                "docId": "kibCorePluginApi",
-                "section": "def-common.SavedObjectReference",
-                "text": "SavedObjectReference"
-              },
+              "SavedObjectReference",
```

Comment:
out of curiosity, what is this change? Was this previously a link to the core plugin and not a link anymore?

Reply:
It's because broken links weren't being removed from children (the function wasn't recursive as it should have been).  I'm actually not sure how it didn't break the build system, but 🤷 .

`SavedObjectReference` is exported from core, but  it's exported from a `types` folder, not `common`, `server` or `client`. This throws the system off. If you go in the docs now, you'll see there are no `Common` types exported in the core API docs, so this was a broken link:

<img width="320" alt="Screen Shot 2021-04-15 at 11 18 26 AM" src="https://user-images.githubusercontent.com/16563603/114894197-6ce14280-9ddc-11eb-829f-109bbd6ecdfb.png">

The core team is planning to move all exports from `types` into `common` so I'm not going to fix the system but wait for them to fix their folder structure.